Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,30 @@
 function sum(elements) {
+    // return 0 if elements is empty. If elements has 1 item, return this item.
+    if (elements.length == 0) {
+        return 0;
+    } else if (elements.length === 1 && typeof elements[0] == "number") {
+        return elements[0];
+    }
+    // declare variables to use it later.
+    let total = 0;
+    let copyElementsStr = [];
+    // check if there is a non-number item inside the elements.
+    for (let i = 0; i < elements.length; i++) {
+        // If item is number, add item to the total array.
+        if (typeof elements[i] === "number") {
+            total += elements[i];
+            // If item is not a number, add item to the copyElementsStr array.
+        } else if (typeof elements[i] !== "number") {
+            copyElementsStr.push(elements[i]);
+        }
+    }
+    // check if given array has only non-number items and return the last item.
+    if (elements.length == copyElementsStr.length) {
```

Comment:
Again, what's the reason to return the last element?

Reply:
Yes, it's the same situation as in max.js. Instead, I should return an error if the value is not numeric. Thank you for your advice.